new-client/src/plugins/Print/PrintModel.js

```diff
@@ -1012,6 +1013,19 @@ export default class PrintModel {
           pdf.setLineWidth(this.margin * 2);
           // Draw the border (margin) around the entire image
           pdf.rect(0, 0, dim[0], dim[1], "S");
+          // If selected as feature in Admin, we draw a frame around the map image
+          if (this.includeImageBorder) {
+            // Frame color is set to dark gray
+            pdf.setDrawColor("#5A5A5A");
```

Comment:
Any reason why the color needs to be hard-coded? Perhaps we could use the same color as text color (which is user-selectable), or allow admins to set the frame color in Admin UI?

Reply:
I think we could use same color as text. Not sure if frame color needs to be selectable. Leaning towards not :)